firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -444,19 +440,19 @@ public void testGetAuthToken_multipleCallsDoNotForceRefresh_fetchedNewTokenOnce(
   public void testGetAuthToken_multipleCallsForceRefresh_fetchedNewTokenTwice() throws Exception {
     when(mockPersistedFid.readPersistedFidEntryValue()).thenReturn(REGISTERED_FID_ENTRY);
     // Use a mock ServiceClient for network calls with delay(1000ms) to ensure first task is not
-    // completed before the second task starts. Hence, we can test multiple calls to getAUthToken()
+    // completed before the second task starts. Hence, we can test multiple calls to getAuthToken()
     // and verify one task waits for another task to complete.
 
     doAnswer(
-            new AnswersWithDelay(
+            AdditionalAnswers.answersWithDelay(
                 1000,
                 new Returns(
```

Comment:
`Returns` is still internal to Mockito. Should be able to use a lambda:

```
doAnswer(AdditionalAnswers.answerWithDelay(1_000, (unused) -> myThing.build()))
```

Reply:
Done. Thanks.